src/components/Highlights/index.tsx

```diff
@@ -6,53 +6,58 @@ import clsx from "clsx"
 
 import styles from "./styles.module.css"
 
-export type Item = {
+export type CardProps = {
 	title: string
 	video: string
 	description: string
 }
 
-export function DisplayItemsHeader({
+function Card(props: CardProps) {
+	return (
+		<li key={props.title} className="card shadow--md">
+			<div className={clsx("card__image")}>
+				<video src={props.video} width="100%" autoPlay controls loop muted></video>
+			</div>
+			<div className="card__body">
+				<Heading as="h4">{props.title}</Heading>
+				<p>{props.description}</p>
+			</div>
+			<ul className={clsx("card__footer")}></ul>
+		</li>
+	)
+}
+
+export function Header({
 	heading,
 	description,
-	link: cta,
+	link,
 }: {
 	heading: string
 	description?: string
-	link: { emoji: string; text: string; href: string }
+	link: { emoji: string; text: string; to: string }
 }) {
 	return (
 		<section className="margin-top--lg margin-bottom--lg text--center">
 			<Heading as="h1">{heading}</Heading>
 			{description && <p>{description}</p>}
-			<Link className={clsx("button button--primary", styles.space)} to={cta.href}>
-				<span aria-hidden="true">{cta.emoji}</span>
-				<span>{cta.text}</span>
+			<Link className={clsx("button button--primary", styles.space)} to={link.to}>
+				<span aria-hidden="true">{link.emoji}</span>
+				<span>{link.text}</span>
 			</Link>
 		</section>
 	)
 }
 
-function DisplayItemsCard(item: Item) {
-	return (
-		<li key={item.title} className="card shadow--md">
-			<div className={clsx("card__image")}>
-				<video src={item.video} width="100%" autoPlay controls loop muted></video>
-			</div>
-			<div className="card__body">
-				<Heading as="h4">{item.title}</Heading>
-				<p>{item.description}</p>
-			</div>
-			<ul className={clsx("card__footer")}></ul>
-		</li>
-	)
-}
-
-export function DisplayItemsList({ items }: { items: Item[] }) {
+export function Cards({ from: cards }: { from: CardProps[] }) {
 	return (
 		<section className="margin-top--lg margin-bottom--xl">
 			<div className="container">
-				<ul className={clsx("container", "clean-list", styles.cards)}>{items.map(item => DisplayItemsCard(item))}</ul>
+				<div className={clsx("margin-bottom--md")}>
+					<Heading as="h2">Our favorites ✨</Heading>
```

Comment:
I'm not a fan of this header...

Reply:
Hmm why? I feel like there could be more padding between the card and the header to enhance visibility while making it more beautiful. I also tried replacing "Our favorites" with `&nbsp;` but it looks a bit strange 🤣

Reply:
Why not just use padding?

Reply:
Meaning that it would look a bit strange to keep the padding without filling in a text like "Our favorites."